module/VuFind/src/VuFind/Auth/Manager.php

```diff
@@ -583,9 +583,12 @@ public function updateEmail(UserRow $user, $email)
         // Depending on verification setting, either do a direct update or else
         // put the new address into a pending state.
         if ($this->config->Authentication->verify_email ?? false) {
-            $user->pending_email = $email;
+            // If new email address is the current address, just reset any pending
+            // email address:
+            $user->pending_email = ($email === $user->email) ? '' : $email;
```

Comment:
@EreMaijala, I simplified this with a ternary operator. Any objections?

Reply:
Nice, no objections!